starboard/nplb/media_set_audio_write_duration_test.cc

```diff
@@ -151,8 +152,9 @@ class SbMediaSetAudioWriteDurationTest
   int index_ = 0;
   int64_t total_duration_ = kDuration;
   // Guard access to |pending_decoder_status_|.
-  mutable SbAtomic32 pending_decoder_status_lock_ =
-      starboard::kSpinLockStateReleased;
+  mutable std::atomic_int pending_decoder_status_lock_ {
```

Comment:
atomic_int32_t?

Reply:
atomic_int32_t is optional type and might be not available for some platforms. I used atomic<int32_t> instead.